<launch>
    <node name="image_pub" pkg="image_processing_hw" type="image_pub_all.py"/>
    <node name="ImageThresholder" pkg="calvin_cv" type="ImageThresholder.py"/>
    <node name="EdgeDetector" pkg="calvin_cv" type="EdgeDetector.py"/>
    <node pkg="rqt_image_view" type="rqt_image_view" name="image_lines_white" args="/image_lines_white"/>
    <node pkg="rqt_image_view" type="rqt_image_view" name="image_lines_yellow" args="/image_lines_yellow"/>
    <node pkg="rqt_image_view" type="rqt_image_view" name="image_edges" args="/image_edges"/>
</launch>
